### ISSUE TITLE
title not found even when there is a title tag and og:title

### ISSUE THREAD

**Issue by alfie-max** [jaimeiniesta/metainspector#191]
Rails 4.2.7.1
Ruby 2.3.1
metainspector (5.2.1)

```
Loading development environment (Rails 4.2.7.1)
>> domain = 'revenew.com'
"revenew.com"
>> MetaInspector.new(domain).to_hash
{
             "url" => "http://www.revenew.com/",
          "scheme" => "http",
            "host" => "www.revenew.com",
        "root_url" => "http://www.revenew.com/",
           "title" => "",
      "best_title" => nil,
     "description" => "",
           "links" => {
        "internal" => [],
        "external" => [],
        "non_http" => []
    },
          "images" => [],
         "charset" => nil,
            "feed" => nil,
    "content_type" => "text/html",
       "meta_tags" => {
              "name" => {},
        "http-equiv" => {},
          "property" => {},
           "charset" => [
            [0] nil
        ]
    },
         "favicon" => nil,
        "response" => {
         "status" => 200,
        "headers" => {
                            "server" => "Sucuri/Cloudproxy",
                              "date" => "Sat, 10 Sep 2016 18:49:15 GMT",
                      "content-type" => "text/html; charset=UTF-8",
                    "content-length" => "16152",
                        "connection" => "close",
                  "content-encoding" => "gzip",
                              "vary" => "Accept-Encoding",
                              "link" => "<http://www.revenew.com/wp-json/>; rel=\"https://api.w.org/\", <http://www.revenew.com/>; rel=shortlink",
                  "x-xss-protection" => "1; mode=block",
                   "x-frame-options" => "SAMEORIGIN",
            "x-content-type-options" => "nosniff",
                       "x-sucuri-id" => "16011"
        }
    }
}

```

**Comment by NeoElit** [comment]
@alfie-max does this library parse gzip compressed responses automatically? the site mentioned by you doesn't seem to respect "accept encoding" header.
It's always sending compressed content.

**Comment by jaimeiniesta** [comment]
Yep, the problem lies in the gzipped response. We don't support it yet, but we can add it.

**Comment by jaimeiniesta** [comment]
This has been fixed in Metainspector 5.3.1, we now automatically deflate gzipped responses.

Thanks for the bug report @alfie-max

**Comment by alfie-max** [comment]
@jaimeiniesta Thanks.
